src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1345,29 +1346,33 @@ void RaftPart::processAskForVoteRequest(const cpp2::AskForVoteRequest& req,
     resp.set_error_code(cpp2::ErrorCode::E_TERM_OUT_OF_DATE);
     return;
```

Comment:
We cannot just return here. When `req.get_term() > term_`，we should also update term_ and step to follower.

Reply:
My bad, hold on

Reply:
My bad, there is a missing local commit. Hold on one second.